testfiles/pgfmanual-en-base-animations.lvt

```diff
@@ -0,0 +1,549 @@
+\documentclass{minimal}
+\input{pgf-regression-test}
+\RequirePackage{fp,pgf,tikz,xcolor}
+
+\usepgfmodule{animations}
+\usetikzlibrary{animations}
+\def\pgfname{\textsc{pgf}}
+
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usetikzlibrary{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usetikzlibrary{views}
+\usepgfmodule{animations}
+\usetikzlibrary{views}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
+\usepgfmodule{animations}
```

Comment:
many duplicates

Reply:
I've just auto-generated these files thus far. They still have to be cleaned and I still have to extract the `graphdrawing` code examples, but they are in the Lua source files not in the pgfmanual.

Reply:
I think in general the examples are likely 'less focussed' than one would write for _new_ tests, but it's a great place to start so this kind of minor inconsistency is fine. Good coverage with possible refactor later sounds like the way to go.